demisto_sdk/commands/split/jsonsplitter.py

```diff
@@ -0,0 +1,125 @@
+                        fg='bright_cyan')
+        self.create_output_dirs()
+        self.create_dashboards()
+        self.create_module()
+        return 0
+
+    def create_output_dirs(self):
+        # no output given
+        if not self.dashboard_dir:
+            # check if in content repository
+            if not is_external_repository() and self.autocreate_dir:
+                pack_name = get_pack_name(self.input)
+                if self.logging:
+                    click.echo(f"No output path given creating dirs in pack {pack_name}")
+                pack_path = os.path.join(PACKS_DIR, pack_name)
+                self.dashboard_dir = os.path.join(pack_path, DASHBOARDS_DIR)
+                self.module_dir = os.path.join(pack_path, GENERIC_MODULES_DIR)
+
+                # create the dirs, dont fail if exist
+                try:
+                    os.mkdir(self.dashboard_dir)
+                except FileExistsError:
+                    pass
+
+                try:
+                    os.mkdir(self.module_dir)
+                except FileExistsError:
+                    pass
+
+            # if not in content create the files locally
+            else:
+                if self.logging:
+                    click.echo("No output path given and not running in content repo, creating files locally")
+                self.dashboard_dir = '.'
+                self.module_dir = '.'
+
+    def create_dashboards(self):
+        if self.logging:
+            click.echo("Starting dashboard creation")
+
+        for view in self.module_json_data.get('views', []):
+            for tab in view.get('tabs', []):
+                dashboard_data = tab.get('dashboard')
+
+                if dashboard_data:
+                    dashboard_file_name = dashboard_data.get('name').replace(' ', '') + '.json'
+                    full_dashboard_path = os.path.join(self.dashboard_dir, dashboard_file_name)
+
+                    if self.logging:
+                        click.echo(f"Creating dashboard: {full_dashboard_path}")
+
+                    with open(full_dashboard_path, 'w') as dashboard_file:
+                        json.dump(dashboard_data, dashboard_file, indent=4)
+
+                    tab['dashboard'] = {'id': dashboard_data.get('id')}
+
+    def create_module(self):
+        if not self.new_module_file:
+            if self.logging:
+                click.echo(f"Updating module file {self.input}")
+            module_file_path = self.input
+
+        else:
+            if self.logging:
+                click.echo("Creating new module file")
+            file_name = os.path.split(self.input)[1]
+            module_file_path = os.path.join(self.module_dir, file_name)
+
+            if os.path.exists(module_file_path):
+                given_name = False
+                while not given_name:
+                    file_name = str(input("\nPlease enter a new file name: "))
```

Comment:
Perhaps add a more clear message? that file already exists, please enter a new file name or something like that. WDYT?

Reply:
Well, it does print `Creating new module file` just before this print so I think it is clear.
Come to think of it my UX thinking here is off, if you have requested to create a new file I should just ask the user for the new file name, I'm currently assuming he'd like to keep the same name and that is not necessarily the case.